features/built_in_matchers/class.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Class matchers
+
+  rspec-expectations a matcher to specify the nature of classes:
+
+    * `expect(Klass).to be_subclass_of(Klazz)`: calls `Klass.ancestors.include?(Klazz)`, which returns true if
+        Klazz is in Klass's class hierarchy or is a module and is included in a class in Klass's
+        class hierarchy. This matcher is aliased as `be_a_subclass_of` and `a_subclass_of`.
```

Comment:
It seems weird to use a bullet list for only one thing.  Can this be combined with the sentence above into a single paragraph?